### PR TITLE
Change Date Added string for Item Details

### PR DESCRIPTION
**Pull request by ThatNerdyPikachu**

### CODE REVIEW

File: src/strings/en-us.json [hunk 1]
```diff
@@ -165,6 +165,8 @@
     "CustomDlnaProfilesHelp": "Create a custom profile to target a new device or override a system profile.",
     "DateAdded": "Date added",
     "DatePlayed": "Date played",
+    "DateAddedColon": "Date added:",
```
Comment (dkanada) [inline src/strings/en-us.json]: It looks like you didn't use this string anywhere.

Comment (ThatNerdyPikachu) [reply src/strings/en-us.json]: Added for redundancy, in case Date Added was used anywhere we want a colon as well.

Comment (joshuaboniface) [reply src/strings/en-us.json]: Why not just put the colon in the in-code string where required? It seems weird to have a single string that exists just to "have a colon at the end".

Comment (ThatNerdyPikachu) [reply src/strings/en-us.json]: That works.

**Pull request closed without merging**